config/locales/de.yml

```diff
@@ -920,7 +920,7 @@ de:
     order_state:
       address: Adresse
       awaiting_return: erwartet Erstattung
-      canceled: Abgebrochen
+      canceled: Storniert
```

Comment:
https://translate.google.com/#auto/de/order%20is%20canceled - in the context of an order, it should be "storniert". Since this is an order state, I do think it's more correct this way. The problem is "cancel" translates to different things depending on context.